native/lambda_game_engine/src/game.rs

```diff
@@ -337,24 +337,19 @@ fn apply_projectiles_collisions(
                     projectile.size,
                     player.size,
                 )
+                && !projectile.attacked_player_ids.contains(&player.id)
             {
                 if player.id == projectile.player_id {
                     continue;
                 }
 
                 player.decrease_health(projectile.damage);
                 player.apply_effects(&projectile.on_hit_effects);
+                projectile.attacked_player_ids.push(player.id);
 
-                projectile.active = false;
-                break;
```

Comment:
I think we need this `break` otherwise the projectile will collide with multiple players if they are in collision range at the same time

(PS: this is more a guess on my side, we should test it out, I just didn't have the time right now)

Reply:
You're right!